src/sentry/snuba/discover.py

```diff
@@ -446,12 +447,12 @@ def get_facets(query, params, limit=20, referrer=None):
     # Resolve the public aliases into the discover dataset names.
     snuba_args, translated_columns = resolve_discover_aliases(snuba_args)
 
-    # Force sampling for more than 9 projects. 9 was chosen arbitrarily.
-    sample = len(snuba_filter.filter_keys["project_id"]) > 9
+    # Force sampling for multi-project results as we don't need accuracy
+    # with that much data.
+    sample = len(snuba_filter.filter_keys["project_id"]) > 2
```

Comment:
Out of curiosity why not `> 1` instead?

Reply:
Sampling + tests results in tests being really flaky and I wanted to have a test using more than one project.